<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YResDial</class>
 <widget class="QWidget" name="YResDial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>130</width>
    <height>110</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>130</width>
    <height>110</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>130</width>
    <height>110</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="res7Button">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>31</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>10 V</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="res5Button">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>0</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>2 V</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="res4Button">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>8</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>1 V</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="MYDial" name="dial">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>21</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="maximum">
    <number>8</number>
   </property>
   <property name="notchesVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="res2Button">
   <property name="geometry">
    <rect>
     <x>-4</x>
     <y>60</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>0.2 V</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="res6Button">
   <property name="geometry">
    <rect>
     <x>83</x>
     <y>8</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>5 V</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="res3Button">
   <property name="geometry">
    <rect>
     <x>-4</x>
     <y>32</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>0.5 V</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="res1Button">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>83</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>0.1 V</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="res9Button">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>83</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>50 V</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="res8Button">
   <property name="geometry">
    <rect>
     <x>101</x>
     <y>60</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>20 V</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MYDial</class>
   <extends>QDial</extends>
   <header>mydial.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
